src/map/spell.cpp

```diff
@@ -163,6 +171,62 @@ bool CSpell::canHitShadow()
     return m_ID != SpellID::Meteor_II && canTargetEnemy();
 }
 
+SPELLFAMILY CSpell::getSpellFamily()
+{
+    // TODO: This should be in the db, hard coded for demo purposes!
```

Comment:
![image](https://user-images.githubusercontent.com/6871475/78782098-71744f00-796f-11ea-89b3-94ed8cb1f919.png)

Reply:
_it is done_